addon/utils/get-asset-map-data.js

```diff
@@ -1,7 +1,5 @@
-import $ from 'jquery';
-
 export default function getAssetMapData() {
-  const assetMapString = $("meta[name='ember-cli-ifa:assetMap']").attr('content');
+  const assetMapString = document.querySelector("meta[name='ember-cli-ifa:assetMap']").content;
```

Comment:
Tried to run `ember s` with this change, it now complains on ember fastboot level, that document is not defined. For fastboot I think we have to come up with something different...

Reply:
That's super disappointing...if we have to stick with jQuery to support fastboot, then so be it, I guess. :\

Reply:
Or just guard against it, as suggested [here](https://github.com/ember-fastboot/ember-cli-fastboot/issues/74#issuecomment-150202828).